.github/workflows/build.yml

```diff
@@ -70,3 +82,7 @@ jobs:
           push: ${{ github.event_name != 'pull_request' && github.repository == 'photoview/photoview' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
+          build-args: |
+            BUILD_DATE=${{ steps.args.outputs.BUILD_DATE }}
+            VERSION=${{ steps.args.outputs.VERSION }}
+            COMMIT_SHA=${{ steps.args.outputs.COMMIT_SHA }}
```

Comment:
Please replace this line with the next one:
`COMMIT_SHA=${{ github.sha }}`
Or if you'd prefer using the Env var (as according to the GitHub docs both ways are supported, however the 1st one looks more consistent visualy):
`COMMIT_SHA=${GITHUB_SHA}`

Reply:
Done.